utils/mock-data-loader/re-insert-mocks.js

```diff
@@ -17,22 +17,34 @@ const { logger, LOGGER_COLOURS } = require('./helpers/logger.helper');
 
 const init = async () => {
   logger.info('REINSERTING MOCKS', { colour: LOGGER_COLOURS.bright });
-
-  const portalToken = await createAndLogInAsInitialUser();
-
-  await cleanAllTables(portalToken);
-  await insertMocks(portalToken);
-  await cleanAllTablesGef(portalToken);
-  await insertMocksGef(portalToken);
-
-  await cleanAllTablesTfm();
-
-  const tfmToken = await createAndLogInAsInitialTfmUser();
-  await insertMocksTfm(tfmToken);
-
-  await deleteInitialTFMUser(tfmToken);
-  await deleteInitialUser(portalToken);
-
+  try {
+    const portalToken = await createAndLogInAsInitialUser();
+
+    await cleanAllTablesPortal(portalToken);
+    await cleanAllTablesGef(portalToken);
+    await cleanAllTablesTfm();
+
+    await insertMocks(portalToken);
+    await insertMocksGef(portalToken);
+
+    const tfmToken = await createAndLogInAsInitialTfmUser();
+    await insertMocksTfm(tfmToken);
+
+    await deleteInitialTFMUser(tfmToken);
+    await deleteInitialUser(portalToken);
+  } catch (error) {
+    logger.warn('An error occurred, attempting to clean all tables');
+    try {
+      const portalToken = await createAndLogInAsInitialUser();
+      await cleanAllTablesPortal(portalToken);
+      await cleanAllTablesGef(portalToken);
+      await cleanAllTablesTfm();
+    } catch {
+      logger.warn('Not all tables could be cleared. Consider manually clearing your database before retrying');
+    }
+    logger.error('The following error occurred while attempting to reinsert mocks:');
```

Comment:
LOC :36 and :45 both being printed out on the console can be confusing, perhaps we should look into merging into one statement please. Thanks

Reply:
image.png

Reply:
![image](https://github.com/UK-Export-Finance/dtfs2/assets/47955140/d1cbee10-cdc3-41fb-b5b3-fad5f7c0b04b)
![image](https://github.com/UK-Export-Finance/dtfs2/assets/47955140/8845cfb5-143a-4ff9-9bd1-dff28c50f0b8)

Reply:
I hope it's not too confusing -- I wanted a clear way of denoting where the clear up begins vs the main script ends after throwing an error... I've updated it to use `error`, so it all should read clearly. Let me know what you think.

For reference these are the logs for a failure:
![image](https://github.com/UK-Export-Finance/dtfs2/assets/47955140/d1cbee10-cdc3-41fb-b5b3-fad5f7c0b04b)

And these are the logs for a failure, and a subsequent failure on the clean up
![image](https://github.com/UK-Export-Finance/dtfs2/assets/47955140/8845cfb5-143a-4ff9-9bd1-dff28c50f0b8)